### ISSUE TITLE
new Device: DPT converter

### ISSUE THREAD

**Issue by farmio** [XKNX/xknx#395]
We could implement a device wich converts payloads from one DPT to another.

Some actuators seem to send percentages as DPT 9 instead of the typical DPT 5 or temperatures as DPT 14 instead of 9.

A converter device could have one input GA and value_type and one output GA and value_type. For every received telegram on the input GA the value would be sent out on the output GA in a different encoding. If the value exceeds the outgoing max/min values or precision we'd need some kind of defined behavior (raise an error or send max/min or round...). 
We could even accept some kind of template language (like HA does - there is for sure a nice python package for this) to do basic transitions of the value like `+ - / *` and `< > =` for DPT 1 output.

What do you think?

**Comment by zaphood1967** [comment]
Yes, this would be a really good idea. Even in the commercial KNX world I have very rarely seen this feature, which could be very helpful indeed
